cmd/zhack.c

```diff
@@ -58,6 +58,12 @@ static importargs_t g_importargs;
 static char *g_pool;
 static boolean_t g_readonly;
 
+typedef enum {
+	ZHACK_REPAIR_OP_UNKNOWN  = (1 << 0),
```

Comment:
This should be set to `0`. Can also implement these as `#defines`, depending on what is more stylistically tasteful.

Reply:
This is how we've handled this elsewhere and is stylistically preferred.

```c
typedef enum {
	ZHACK_REPAIR_OP_UNKNOWN  = 0,
	ZHACK_REPAIR_OP_CKSUM    = (1 << 0),
	ZHACK_REPAIR_OP_UNDETACH = (1 << 1),
} zhack_repair_op_t;
```